Remove policy samples

dockerymick opened the pull request:
https://github.com/open-cluster-management/backlog/issues/8269

dockerymick committed:
added link refernces to the policies in policy-collection

dockerymick committed:
made suggesstion and repurposed one link until agreed upon

dockerymick committed:
added reference links

dockerymick committed:
removed the role policy sample

dockerymick committed:
other changes

dockerymick commented on security/config_policy_ctrl.adoc:
```diff
@@ -124,9 +124,6 @@ The `exclude` parameter specifies the namespaces you explicitly do not want to a
 `mustnothave`: Indicated that an object with the same name or labels cannot exist and need to be deleted, regardless of the specification or rules.
 |===
 
+See configuration policies samples that are supported by {product-title-short} from the link:https://github.com/open-cluster-management/policy-collection/blob/master/stable/SC-System-and-Communications-Protection/policy-certificate.yaml[CM-Configuration-Management folder]. Learn about how policies are applied on your hub cluster, see xref:../security/policy_sample_intro.adoc#policy-samples[Policy ...
```
See the policy examples that use the NIST

dockerymick committed:
removed yaml sample for cert

dockerymick committed:
removed sample from IAM and referenced the open repo

dockerymick committed:
reference open repo fo rpolicy sample

dockerymick committed:
removed policy sample and referenced the open repo

dockerymick committed:
small change

dockerymick committed:
improved links, updated doc title, and reference open repo

dockerymick committed:
there were a lot of changes in this commit

dockerymick committed:
updating role binding the links

dockerymick committed:
last few policy samples removed

dockerymick committed:
Merge branch '2.2_stage' into 8269

dockerymick commented:
@ycao56 apologies for the long PR, I ran the spellcheck tool and decided to update Rolebinding to role binding. You can ignore the files related to that change. If it is ok with you, I would like to merge this PR and then I will address the comments you have made in the google doc. let me know what you think or prefer

cadawson commented on security/create_rb_policy.adoc:
```diff
@@ -72,30 +72,30 @@ Complete the following steps to create a rolebinding policy from the console:
 . Log in to your cluster from the console.
 . From the navigation menu, click *Governance and risk* to view a table list of your policies.
 +
 NOTE: You can filter the table list of your policies by selecting the _Policies_ tab or _Cluster violations_ tab.
```
*Note:*

cadawson commented on security/policy_sample_intro.adoc:
```diff
@@ -1,25 +1,25 @@
+* xref:../security/namespace_policy.adoc#namespace-policy[Namespace policy]
+* xref:../security/pod_nginx_policy.adoc#pod-nginx-policy[Pod nginx policy]
+* xref:../security/psp_policy.adoc#pod-security-policy[Pod security policy]
+* link:../security/role_policy.adoc#role-policy[Role policy]
```
Will this work? Let me know if it does...

dockerymick replied:
I believe it will, i didn't notice it was like that tho. Thanks. I will maintain consistency and change to xref

cadawson commented on security/psp_policy.adoc:
```diff
@@ -99,53 +99,4 @@ Used to list any other Kubernetes object that must be evaluated or applied to th
-
-See xref:../security/create_psp_policy.adoc#managing-pod-security-policies[Managing pod security policies] for more information.
-View other configuration policies that are monitored by controller, see the xref:../security/config_policy_ctrl.adoc#kubernetes-configuration-policy-controller[Kubernetes configuration policy controller] page.
+See link:https://github.com/open-cluster-management/policy-collection/blob/master/stable/SI-System-and-Information-Integrity/policy-psp.yaml[`policy-psp.yaml`] to view the sample policy. View xref:../security/create_psp_policy.adoc#managing-pod-security-policies[Managing pod security policies] for more information. Refer to xref:../security/config_policy_ctrl.adoc#kubernetes-configuration-policy- ...
```
...monitored by the controller.?

cadawson commented on security/scc_policy.adoc:
```diff
@@ -108,113 +105,10 @@ For explanations on the contents of a SCC policy, see link:https://docs.openshif
-Apply an Security context constraints (SCC) policy to control permissions for pods by defining conditions in the policy.
-For more information see, link:https://docs.openshift.com/container-platform/4.3/authentication/managing-security-context-constraints.html#security-context-constraints-about_configuring-internal-oauth[Managing Security Context Constraints (SCC)].
-Your SCC policy might resemble the following YAML file:
+Apply an Security context constraints (SCC) policy to control permissions for pods by defining conditions in the policy. For more information see, link:https://docs.openshift.com/container-platform/4.3/authentication/managing-security-context-constraints.html#security-context-constraints-about_configuring-internal-oauth[Managing Security Context Constraints (SCC)].
```
Apply a Security context constraints (SCC) policy...

cadawson replied:
Can we bump this link up to a later version of OCP? I verified that the same link exists for 4.4, 4.5, and 4.6. All you have to do is change the number in the URL. The reason is that 4.3 is going out of support.

cadawson reviewed (changes requested):
Nice job! Looks like the links all work well. I added a few things to look at. After that, it looks good. Thanks!

dockerymick committed:
Update create_rb_policy.adoc

update after Chris' review

dockerymick committed:
Update policy_sample_intro.adoc

Update after Chris' review

dockerymick committed:
Update psp_policy.adoc

updated the indentation for the policy and made a change according to Chris' review

dockerymick committed:
Update scc_policy.adoc

changes after Chris' review

dockerymick commented:
@cadawson thank you very much for your review, your feedback was very helpful. I have addressed your comments and will continue to merge this PR
